src/index.js

```diff
@@ -0,0 +1,63 @@
+import './style.css';
+import {
+  FnToDoList,
+  addTask,
+  editTask,
+  deleteTask,
+} from './modules/UI.js';
+import clearCompleted from './modules/clearCompleted.js';
+import markTask from './modules/marktask.js';
+import { updateLocalStorage, getLocalStorage } from './modules/storage.js';
+
+const input = document.querySelector('.input');
+const List = document.querySelector('.list-ul');
+const clearCompletedBtn = document.querySelector('.clear-btn');
+const addTaskBtn = document.querySelector('.add-btn');
+
+let Tasks = getLocalStorage();
+
+FnToDoList(Tasks);
+
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
+    addTask(Tasks, input.value);
+    input.value = '';
+    updateLocalStorage(Tasks);
+    FnToDoList(Tasks);
+  }
+});
+
+addTaskBtn.addEventListener('click', () => {
+  if (input.value !== '') {
+    addTask(Tasks, input.value);
+    input.value = '';
+    updateLocalStorage(Tasks);
+    FnToDoList(Tasks);
+  }
+});
+
+List.addEventListener('click', (e) => {
+  if (e.target.closest('.list-li-checkbox')) {
+    markTask(e, Tasks);
+  }
+  updateLocalStorage(Tasks);
+  FnToDoList(Tasks);
+});
+
```

Comment:
[] Instead of using 'e', please use proper naming like event, to make it easy to read and your code professional.

Reply:
@SaneMyburg Thank You for your time and valuable suggestions.